libs/api/jest.config.ts

```diff
@@ -15,5 +15,6 @@ export default {
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
   coverageDirectory: '../../coverage/libs/api',
   setupFilesAfterEnv: ['./__tests__/setup.ts'],
-  globalSetup: '<rootDir>/__tests__/setup-database.js'
+  globalSetup: '<rootDir>/__tests__/setup-database.js',
+  collectCoverageFrom: ['!**/*']
```

Comment:
code coverage is super slow on this, i guess because of the way graphql is tested